spring-integration-file/src/main/resources/org/springframework/integration/file/config/spring-integration-file-4.2.xsd

```diff
@@ -349,105 +349,125 @@ Only files matching this regular expression will be picked up by this adapter.
+    		<xsd:annotation>
+    			<xsd:documentation><![CDATA[
 	Specify whether to delete source files after writing to the destination directory.
 	This will take effect if the Message payload is the actual source File instance
 	or if the original File instance (or its path) is available in the header value
 	associated with the FileHeaders.ORIGINAL_FILE constant. The default value is false.
 				]]></xsd:documentation>
-            </xsd:annotation>
-        </xsd:attribute>
-        <xsd:attribute name="order" type="xsd:string">
-            <xsd:annotation>
-                <xsd:documentation><![CDATA[
+    		</xsd:annotation>
+    	</xsd:attribute>
+    	<xsd:attribute name="order" type="xsd:string">
+    		<xsd:annotation>
+    			<xsd:documentation><![CDATA[
 	Specifies the order for invocation when this endpoint is connected as a
 	subscriber to a SubscribableChannel.
 				]]></xsd:documentation>
-            </xsd:annotation>
-        </xsd:attribute>
-        <xsd:attribute name="auto-create-directory" type="xsd:string" default="true">
-            <xsd:annotation>
-                <xsd:documentation><![CDATA[Specify whether to automatically create the destination directory if it does not yet exist when this
+    		</xsd:annotation>
+    	</xsd:attribute>
+    	<xsd:attribute name="auto-create-directory" type="xsd:string"
+    		default="true">
+    		<xsd:annotation>
+    			<xsd:documentation><![CDATA[Specify whether to automatically create the destination directory if it does not yet exist when this
                     adapter is being initialized. The default value is 'true'. If set to 'false' and the directory does
                     not exist upon initialization, an Exception will be thrown.]]>
-                </xsd:documentation>
-            </xsd:annotation>
-        </xsd:attribute>
-        <xsd:attribute name="charset" type="xsd:string">
-            <xsd:annotation>
-                <xsd:documentation>
-                    <![CDATA[Set the charset name to use when writing a File from a String-based
+    			</xsd:documentation>
+    		</xsd:annotation>
+    	</xsd:attribute>
+    	<xsd:attribute name="charset" type="xsd:string">
+    		<xsd:annotation>
+    			<xsd:documentation>
+    				<![CDATA[Set the charset name to use when writing a File from a String-based
                     Message payload, e.g. charset="UTF-8". If not set, the default charset of this
                     Java virtual machine is used.
                     ]]>
-                </xsd:documentation>
-            </xsd:annotation>
-        </xsd:attribute>
-		<xsd:attributeGroup ref="integration:smartLifeCycleAttributeGroup"/>
+    			</xsd:documentation>
+    		</xsd:annotation>
+    	</xsd:attribute>
+    	<xsd:attribute name="append-new-line" type="xsd:string" default="false">
+    	    <xsd:annotation>
+    			<xsd:documentation>
+    				<![CDATA[Set to 'true' to append a new-line after each write (only applicable if mode='APPEND'). 
```

Comment:
`only applicable...` isn't relevant anymore. See the final solution from the related PR.
And, please, revert any other formatting changes. It's very difficult to determine exactly your changes.